<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>494</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>EAR Audio Rectifier</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Pics.qrc">
    <normaloff>:/Logo.png</normaloff>:/Logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEqualizers">
    <property name="title">
     <string>Equalizers</string>
    </property>
    <addaction name="actionSaveLeft"/>
    <addaction name="actionSaveRight"/>
    <addaction name="separator"/>
    <addaction name="actionLoadLeft"/>
    <addaction name="actionLoadRight"/>
   </widget>
   <widget class="QMenu" name="menuMeasure">
    <property name="title">
     <string>Calibrate</string>
    </property>
    <addaction name="calibrateAction"/>
   </widget>
   <addaction name="menuEqualizers"/>
   <addaction name="menuMeasure"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="controlDockWidget">
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>455</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>250</width>
     <height>800</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Control Bar</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Source:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="signalSourceComboBox"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Left Channel:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="musicSourceLeftChannelLevel">
          <property name="value">
           <number>24</number>
          </property>
          <property name="textVisible">
           <bool>false</bool>
          </property>
          <property name="format">
           <string>Left (%p%)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Right Channel:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="musicSourceRightChannelLevel">
          <property name="value">
           <number>24</number>
          </property>
          <property name="textVisible">
           <bool>false</bool>
          </property>
          <property name="format">
           <string>Right (%p%)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Microphone</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Left Channel:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="microphoneLeftChannelLevel">
          <property name="value">
           <number>24</number>
          </property>
          <property name="textVisible">
           <bool>false</bool>
          </property>
          <property name="format">
           <string>Left (%p%)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Right Channel:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="microphoneRightChannelLevel">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="value">
           <number>24</number>
          </property>
          <property name="textVisible">
           <bool>false</bool>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="format">
           <string>Right (%p%)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="bypass">
       <property name="text">
        <string>Bypass</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="automaticAdaption">
       <property name="text">
        <string>Automatic adaption</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetControlsPushButton">
       <property name="text">
        <string>Reset Controls</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>19</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionSaveLeft">
   <property name="text">
    <string>Left -&gt; File</string>
   </property>
  </action>
  <action name="actionSaveRight">
   <property name="text">
    <string>Right -&gt; File</string>
   </property>
  </action>
  <action name="actionLoadLeft">
   <property name="text">
    <string>File -&gt; Left</string>
   </property>
  </action>
  <action name="actionLoadRight">
   <property name="text">
    <string>File -&gt; Right</string>
   </property>
  </action>
  <action name="calibrateAction">
   <property name="text">
    <string>Calibrate Latency</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../Pics.qrc"/>
 </resources>
 <connections/>
</ui>
